pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -547,11 +548,15 @@ type BootMenu struct {
 }
 
 // TODO <loader readonly='yes' secure='no' type='rom'>/usr/lib/xen/boot/hvmloader</loader>
-type BIOS struct {
+type Loader struct {
+	ReadOnly bool   `xml:"readonly,attr,omitempty"`
```

Comment:
Error: "unknown readonly value: true". It should be "yes" or "no": https://libvirt.org/formatdomain.html#elementsOSBIOS

```
{"component":"virt-launcher","kind":"","level":"error","msg":"Defining the VirtualMachineInstance failed.20, Message='unknown readonly value: true')","timestamp":"2018-12-10T20:59:01.339237Z","uid":"323d8137-fc
{"component":"virt-launcher","level":"error","msg":"unknown readonly value: true","pos":"virDomainLoaderD
{"component":"virt-launcher","kind":"","level":"error","msg":"Failed to sync vmi","name":"testvm","namespue: true')","timestamp":"2018-12-10T20:59:01.339292Z","uid":"323d8137-fcbe-11e8-bde6-34e6d78281c3"}
```